README.md

```diff
@@ -26,28 +26,72 @@ Set up UI for Apache Kafka with just a couple of easy commands to visualize your
 # The Interface
 UI for Apache Kafka wraps major functions of Apache Kafka with an intuitive user interface.
 
-![Interface](images/Interface.gif)
-
 ## Topics
 UI for Apache Kafka makes it easy for you to create topics in your browser by several clicks, 
 pasting your own parameters, and viewing topics in the list.
 
-![Create Topic](https://github.com/provectus/kafka-ui/blob/6d1219c70e9e24e52db64936e1a5d5ada189fecc/images/Create_topic_kafka-ui.gif)
+![Create Topic](images/Create_topic_kafka-ui.gif)
+
+For more comfortable work you can navigate from connectors to consumers through their topics, ability to read and write to all of the topics used by the 
+connectors, overview topic`s settings. 
+
+![Connector_Topic_Consumer](images/Connector_Topic_Consumer.gif)
 
 ### Messages
 Let's say we want to produce messages for our topic. With the UI for Apache Kafka we can send or write data/messages to the Kafka topics without effort by specifying parameters, and viewing messages in the list.
 
-![Produce Message](https://github.com/provectus/kafka-ui/blob/8ecc012e6ce2b5651bdd5fef9056cc8edd9b41bf/images/Create_message_kafka-ui.gif)
+![Produce Message](images/Create_message_kafka-ui.gif)
 
 ## Schema registry
 There are 3 supported types of schemas: Avro®, JSON Schema, and Protobuf schemas.
 
 ![Create Schema Registry](images/Create_schema.gif)
 
+Before sending avro messages to the topic, you have to register avro schema for the topic to the schema registry. Now all these steps are easy to do 
+with a few clicks in a user-friendly interface.
+
+![Avro Schema Topic](images/Schema_Topic.gif)
+
 # Getting Started
 
 To run UI for Apache Kafka, you can use a pre-built Docker image or build it locally.  
 
+## Configuration
+
+We have plenty of [docker-compose files](guides/yaml-description.md) as examples. Please check them out in ``docker`` directory.
```

Comment:
The second sentence is no more needed since we're pointing to a file which contains all the links for the compose files.
```suggestion
We have plenty of [docker-compose files](guides/yaml-description.md) as examples. They're built for various configuration stacks.
```